src/schema/mutation/editDashboard.mutation.ts

```diff
@@ -23,6 +23,7 @@ export default {
     structure: { type: GraphQLJSON },
     name: { type: GraphQLString },
     showFilter: { type: GraphQLBoolean },
+    buttons: { type: GraphQLJSON },
```

Comment:
@matheus-relief 
there we can create a custom input type

Reply:
done